docs/docs/01-ibc/03-apps/00-ibcv2apps.md

```diff
@@ -0,0 +1,307 @@
+---
+title: IBC v2 Applications
+sidebar_label: IBC v2 Applications
+sidebar_position: 0
+slug: /ibc/apps/ibcv2apps
+---
+:::note Synopsis
+Learn how to implement IBC v2 applications
+:::
+
+To build an IBC v2 application the following steps are required:
+
+1. [Implement the `IBCModule` interface](#implement-the-ibcmodule-interface)
+2. [Bind Ports](#bind-ports)
+3. [Implement the IBCModule Keeper](#implement-the-ibcmodule-keeper)
+4. [Implement application payload and success acknowledgement](#packets-and-payloads)
+5. [Set and Seal the IBC Router](#routing)
+
+## Implement the `IBCModule` interface
+
+The Cosmos SDK expects all IBC modules to implement the [`IBCModule`
```

Comment:
Maybe worth mentioning that this Interface is distinct from the porttypes.IBCModule?